roles/deprovision-unifiedpush-apb/tasks/main.yml

```diff
@@ -3,7 +3,7 @@
     namespace: '{{ namespace }}'
     state: absent
 
-- openshift_v1_deployment_config:
+- k8s_extensions_v1beta1_deployment:
```

Comment:
As we're using `Deployment` instead of `DeploymentConfig` now, it broke the "checker" here: https://github.com/aerogearcatalog/unifiedpush-apb/pull/9/files#diff-bc1ad74da0ee962b50fe4d238c15891dR103

I tried to replace it with `--selector="deployment=mysql"`, but that does not work with the selector. We can easily fix that with `grep`, if there's no other workaround.

Reply:
Using `--selector="service=mysql"` should work.